app/src/main/java/io/homeassistant/companion/android/notifications/NotificationsDB.kt

```diff
@@ -0,0 +1,106 @@
+package io.homeassistant.companion.android.notifications
```

Comment:
This should probably go in data with all the other persisting stuff?

Reply:
Will do.